.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "es6": true
+    },
+    "extends": ["eslint:recommended"],
```

Comment:
NAB: should consider and discuss other extensions, personally I use airbnb-base and prettier which are very opinionated.

Reply:
I work more with PHP than with Javascript. Therefore, I do not know the most extensions and standards. You are free to create a pull request if you can optimize something. ;-)